### PR TITLE
Header and Logo

### PR DESCRIPTION
**Pull request by SlowV**
a xem được chưa

### CODE REVIEW

File: header.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+						<span class="bIcon">3</span>
+						<span class="text-bicon">&nbsp;Bấm  Thánh toán &nbsp;&nbsp;&nbsp;</span>
+						<span class="glyphicon glyphicon-chevron-right">&nbsp;&nbsp;</span>
+					</li>
+					<li>
+						<span class="bIcon">4</span>
+						<span class="text-bicon">&nbsp;Shop sẽ  giao hàng ngay</span>
+					</li>
+					<div>&nbsp;</div>
+					<li>
```
[the remark this answers is not in the document]
Comment (tungntd00605) [reply header.html]: căn chỉnh lại sao cho phần chữ địa chỉ và cái hình giỏ hàng nó nằm ngang nhau và chính giữa, nên căn chỉnh lại css của tag "<li" cho chữ và ảnh nó nằm giữa dòng.
Cái giỏ hàng thì ko nên đặt vào tab <a, dùng tag <div để nó ko bị khoảng trắng nhảy dòng

File: header.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+						<span class="bIcon">3</span>
+						<span class="text-bicon">&nbsp;Bấm  Thánh toán &nbsp;&nbsp;&nbsp;</span>
+						<span class="glyphicon glyphicon-chevron-right">&nbsp;&nbsp;</span>
+					</li>
+					<li>
+						<span class="bIcon">4</span>
+						<span class="text-bicon">&nbsp;Shop sẽ  giao hàng ngay</span>
+					</li>
+					<div>&nbsp;</div>
```
Comment (tungntd00605) [inline header.html]: Không dùng các div thế này để căn chỉnh nhé, dùng Margin hoặc Padding để chỉnh nó

File: header.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+							01469555602  -  0962127341
+						</span>
+						<span class="cartIcon">
+							<a href="#">
+								<img src="img/cartIcon.png" class=" col-md-3 col-xs-12 col-sm-2" width="100%">
+							</a>
+						</span>
+					</li>
+				</ul>
+				<div>&nbsp;</div>
```
Comment (tungntd00605) [inline header.html]: Không dùng các div thế này để căn chỉnh nhé, dùng Margin hoặc Padding để chỉnh nó

File: header.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+				<span class="col-md-5 col-sm-10 col-xs-12" style="float: left;">
```
Comment (tungntd00605) [inline header.html]: - Những phần nằm trong cùng 1 khối cùng dòng thì cho vào 1 cái div, có thể cho attribute của nó là "row" của bootstrap để mình căn float cho chuẩn.
- Xem lại về responsive cho phần này ở các độ phân giải nhỏ hơn sao cho phù hợp

File: header.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+					<a href="#">
+						<img src="img/signup_icon.png" class="img-rounded">
+					</a>
+				</span>
+			</div>
+		</div>
+	</div>
+</section>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (tungntd00605) [inline header.html]: Nên download jquery về và nhét trong folder nhé đừng để link thế này nó tốn thêm tài nguyên user down về
